CasaCatalog/app/src/main/res/layout/activity_add_item.xml

```diff
@@ -1,301 +1,396 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Constraint Layout needs to be restored. It's helps the photo delete button remain on the screen. Adding notes in case someone else goes to merge this in.